java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/JUnit5Runner.java

```diff
@@ -34,7 +38,7 @@ public static void main(String[] args) {
       var constructor =
           Class.forName(JUNIT5_RUNNER_CLASS).asSubclass(RunsTest.class).getConstructor();
       var runsTest = constructor.newInstance();
-      if (!runsTest.run(testSuite)) {
+      if (!runsTest.run(testSuite, includeEngines, excludeEngines)) {
```

Comment:
Which means that all we need to do is pass in the `testSuite`. These other two parameters can be dropped